SDU-Sem3-G1-NES/Project
# Implemented Krzysztof's API in docker compose using a watchdog to mon…

…itor for file changes

Steps to test:

1. copy the new EXAMPLE.env to .env and set all your variables. You can use the default API port. 
2. `docker compose up --build -d`
3. `docker compose logs -f linaksimulatorapi`
4. Do any arbitrary change to api.py (add an empty line), save it, and verify that the watcher restarts the script.
5. Visit this URL in a browser and make sure the API returns the tables: [http://127.0.0.1:8000/api/v1/E9Y2LxT4g1hQZ7aD8nR3mWx5P0qK6pV7/desks](http://127.0.0.1:8000/api/v1/E9Y2LxT4g1hQZ7aD8nR3mWx5P0qK6pV7/desks)
